docs/README.md

```diff
@@ -19,417 +17,498 @@
 
-There are several areas in which we wish to automate for efficiency: consistency, workflow,  quality and statistics. The basic features for these areas are as follows:
+And for issues:
+`issues.opened`, `issues.edited`, `issues.labeled`, `issues.unlabeled`, `issues.milestoned`, `issues.demilestoned`, `issues.assigned`, `issues.unassigned`, `issues.synchronize`
 
-### Consistency
+>  ☝ **NOTE:** More details about events can be found on the [GitHub events page](https://developer.github.com/v3/activity/events).
 
-- *Pull Request* validation for standards based on configured rulesets. ![completed](https://img.shields.io/badge/Status-completed-green.svg)
+For convenience, wildcards can be used: `pull_request.*`, `issues.*`, `pull_request_review.*`
 
-- Notify by creating a comment in *Issues* that do not adhere to configured ruleset. ![completed](https://img.shields.io/badge/Status-completed-green.svg)
 
-- Ruleset across Repos. As a way to enforce and encourage standards in an organization or guide team members on the organizations' engineering best practises.
+> ☝ **NOTE:**  Each `validator` and `action` declares it's own supported events. Read the [validator](#validators) and [action](#actions) sections to find out which events are supported respectively.
 
-- Repo(s) audit. Scan repo(s) for standards configured in rulesets including existance OWNERS file, or .github contains TEMPLATES. Notify through creation of an issue in the repo(s).
+## Validators
 
-### Workflow
+### approvals
 
-- Kanban WIP limits. Limit the number of open Pull Requests by author.
+```yml
+- do: approvals
+  min:
+    count: 2 # Number of minimum reviewers. In this case 2.
+    message: 'Custom message...'
+  required:
+    reviewers: [ user1, user2 ]   # list of github usernames required to review
+    owners: true # accepts boolean. When true,  the file .github/CODEOWNER is read and owners made required reviewers
+    message: 'Custom message...'
+```
 
-- Staleness and Reminders: create a comment on any *Issue* or *Pull Request* found to be stale such that the author and collaborators are notified. ![completed](https://img.shields.io/badge/Status-completed-green.svg)
+Supported events:
 
-- Projects WIP Limits by column in GitHub Projects.
+```js
+'pull_request.*', 'pull_request_review.*'
+```
 
-- Better GitHub Projects automation. i.e. When an issue is assigned, the card automatically moves from column 1 (i.e. `Backlog` if configured as such) to column 2 (i.e. `In Progress` if configured as such) in the kanban board.
+### assignee
 
-- Improved Project Management Integrations (two way): Clubhouse, Pivotal Tracker, Jira, Trello
+```yml
+- do: assignee
+  max:
+    count: 2 # maximum number of assignees
+    message: 'test string'
+  min:  
+    count: 2 # min number of assignees
+    message: 'test string'
```

Comment:
same here as above , `There should be at least 2 assignees`

Reply:
Good suggestion. I updated with your proposal.